### PR TITLE
TabPageHeader redesign + small fixes

### PR DESCRIPTION
**Pull request by Morpog**

### CODE REVIEW

**Comment by Morpog** [comment]
Unfortunately it does not make sense to show content behind TabPageHeader, because I would need to add a margin at bottom in SlideShowView with same height as TabPageHeader, so that all content can be scrolled above the TabPageHeader. Maybe someone else can find a solution for that?

File: qml/tweetian-harmattan/Component/TabPageHeader.qml [hunk 1]
```diff
@@ -38,10 +38,17 @@ Item {
         anchors.fill: parent
         contentHeight: parent.height
 
+        Rectangle {
+            id: backgroundColor
+            anchors.fill: parent
+            color: Qt.darker(Theme.highlightBackgroundColor, 1.8)
```
Comment (siteshwar) [inline qml/tweetian-harmattan/Component/TabPageHeader.qml:44]: I find this color a bit odd in the tab headers (may be it's just me).

**Comment by veskuh** [comment]
I don't like how this looks in normal situation. It makes the tab header too prominent. I  like current look of tab page header. It is there but it does not draw attention, and the focus is on the content.

This does solve the issue of content showing up behind menu when push up menu is open. I have found an alternative solution (exposing the true height from TabHeader and using it for mainView) for that problem which I'm going to use instead of this.

Tab header height I'll change to suggested one.

**Comment by Morpog** [comment]
Yeah, without the content behind it looks a bit out of place. Also it needs a bit getting used to. As I liked the old behavior too I'm closing this and checking out your commits. Will do a follow up with the small fixes and icons.

**Pull request closed without merging**